model/src/application/component/mod.rs

```diff
@@ -156,7 +151,6 @@ impl<'de> Visitor<'de> for ComponentVisitor {
                     // Encountered when we run into an unknown key.
                     map.next_value::<IgnoredAny>()?;
 
-                    #[cfg(feature = "tracing")]
                     tracing::trace!("ran into an unknown key: {:?}", _why);
```

Comment:
And in match statement
```suggestion
                    tracing::trace!("ran into an unknown key: {:?}", why);